pillar.example

```diff
@@ -47,3 +47,16 @@ zookeeper:
 # zookeeper:
 #   config:
 #     bind_address: 0.0.0.0
+
+# If you want to provide your own list of zookeeper nodes you can use zookeeper:nodes parameter. 
+# In this case targeting method will not be used.
+# As a node identifier you can use hostname, IP address, fqdn, minion id.
+# You cannot use an IP address of the proxy server which redirects requests to the zookeeper node. 
+# You can use hostname of the proxy server which redirects requests to the zookeeper node only if 
+# zookeeper node has the same hostname as the proxy does.
```

Comment:
Maybe this text also could be putted to the README file? Were we already have described the options of how to build a cluster.